### PR TITLE
Fix no system role for SLED

### PR DESCRIPTION
**Pull request by okurz**
Replacing leanos as that product does not exist anymore.

### CODE REVIEW

File: lib/registration.pm [hunk 1]
```diff
@@ -116,7 +116,11 @@ sub fill_in_registration_data {
         if (match_has_tag 'bsc#1056413') {
             record_soft_failure('bsc#1056413');
             # Add expected modules to select them manually, as not preselected
-            my $addons = 'base,script,desktop,serverapp' . (get_var('SCC_ADDONS') ? ',' . get_var('SCC_ADDONS') : '');
+            my %addons = (
+                sles => 'base,script,desktop,serverapp',
+                sled => 'base,script,desktop'
```
Comment (coolo) [inline lib/registration.pm]: sled also gets desktop productivity - however you appr it.

Comment (okurz) [reply lib/registration.pm]: "appr"? There is no module "desktop productivity" yet

Comment (coolo) [reply lib/registration.pm]: there is one - possibly not in scc though

Comment (okurz) [reply lib/registration.pm]: yes, I know. That doesn't help

**Comment by okurz** [comment]
I guess "you appr" means the equivalent of LGTM

**Pull request merged**